.gitignore

```diff
@@ -8,3 +8,4 @@ MANIFEST
 tests/regressiontests/outputs
 tests/.coverage
 tests/.coveralls.yml
+.idea
```

Comment:
This is something that should not be defined at the project level but in your own environment. 
https://help.github.com/articles/ignoring-files/#create-a-global-gitignore